addons/udes_stock/views/stock_picking.xml

```diff
@@ -158,6 +158,30 @@
                 <attribute name="attrs">{'readonly': [('state', '!=', 'draft')],  'required': [('picking_type_code', '=', 'incoming')]}</attribute>
             </xpath>
 
+            <xpath expr="//field[@name='date_done']" position="attributes">
+                <attribute name="attrs">{'invisible': [('state', 'not in', ('done', 'cancel'))]}</attribute>
+                <attribute name="string">Date of Transfer</attribute>
+            </xpath>
+
+        </field>
+    </record>
+
+
+    <record id="vpicktree" model="ir.ui.view">
```

Comment:
Suggestion: Add a comment above each override to explain why we are making the changes.